features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/OtelTracerProvider.kt

```diff
@@ -134,6 +136,15 @@ class OtelTracerProvider(
                     { TRACING_NOT_ENABLED_ERROR_MESSAGE }
                 )
             }
+            val rumFeature = sdkCore.getFeature(Feature.RUM_FEATURE_NAME)
+            if (bundleWithRumEnabled && rumFeature == null) {
+                sdkCore.internalLogger.log(
+                    InternalLogger.Level.WARN,
+                    InternalLogger.Target.USER,
+                    { RUM_NOT_ENABLED_ERROR_MESSAGE }
+                )
+                bundleWithRumEnabled = false
+            }
```

Comment:
IMO we need to revisit this logic (I know it is already exists for `AndroidTracer`): in theory with v2 we can have things enabled independently, so nothing in the setup prevents enabling RUM after Tracer is enabled. IMO this shouldn't prevent us to do bundling with RUM. Is such logic exist on iOS as well?

Reply:
Hmmm...good point. I just wanted to follow what was already in the `AndroidTracer` but it makes sense maybe to remove it. Waiting for @xgouchet opinion here also.

Reply:
I agree we need to evaluat the bundling at span creation as the RUM feature can be enabled/disabled independently

Reply:
yes agree,  we are already checking and warning the user at the spanbuilder decoration level so there is no need for other log for now. I will just remove this from here and add a ticket to remove it also for the `AndroidTracer`.